crates/nu-protocol/src/config/table.rs

```diff
@@ -3,7 +3,7 @@ use crate::{record, Config, ShellError, Span, Value};
 use serde::{Deserialize, Serialize};
 use std::str::FromStr;
 
-#[derive(Serialize, Deserialize, Clone, Copy, Debug, Default)]
+#[derive(Serialize, Deserialize, Clone, Copy, Debug)]
```

Comment:
I am not saying we should get rid of `impl Default`.

If there is a notion of a sane default this is always great to implement not just to please the [Rust API guidelines gods](https://rust-lang.github.io/api-guidelines/interoperability.html?highlight=default#types-eagerly-implement-common-traits-c-common-traits) but also because `T: Default` unlocks some helpful safe-Rust patterns and methods on data-structures. Prime example being [`std::mem::take()`](https://doc.rust-lang.org/std/mem/fn.take.html). 

I am rather inclined to say we don't need an additional `Default` variant, if we don't plan to at runtime change the meaning of `TableMode::Default`.

Reply:
My mistake, you said "move #[default]" and I heard "remove #[default]". Hopefully the latest push is better.